mw-webapp/cypress/e2e/home.cy.ts

```diff
@@ -0,0 +1,16 @@
+import pageData from "../fixtures/homeFixture.json";
+import {homeSelectors} from "cypress/scopesSelectors/homeSelectors";
+
+describe('Home page scope tests', () => {
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('homePage_Open', () => {
+    homeSelectors.pageTitle.getTitle().should('contain', pageData.title);
+    homeSelectors.items.getStartBtn().should('have.text', pageData.btnTitle);
```

Comment:
PLease don't use short words (btn, lvl etc). Its not clear what they mean. Its just few symbols but code will be more clear

Reply:
Ok.